src/index.html

```diff
@@ -100,7 +100,21 @@
                         <li class="menu-item-has-children">
                             <a href="index.html">Home</a>
                         </li>
-
+                        <li class="menu-item-has-children">
+                            <a href="">Why us?</a>
+                        </li>
+                        <li class="menu-item-has-children">
+                            <a href="">Features</a>
+                        </li>
+                        <li class="menu-item-has-children">
+                            <a href="">Blog</a>
+                        </li>
+                        <li class="menu-item-has-children">
+                            <a href="">Pricing</a>
+                        </li>
```

Comment:
Shall we keep this commented until we implement those pictures or link them to the related sections in current page